vendor/github.com/percona/percona-toolkit/src/go/mongolib/proto/system.profile.go

```diff
@@ -178,7 +178,7 @@ func (self ExampleQuery) ExplainCmd() bson.D {
 					} else {
 						cmd = append(cmd[:i], cmd[i+1:]...)
 					}
-					break
+					// break LOOP
```

Comment:
Was this `break` removed with intention?

Reply:
yes. it was unused and highlighted by the linter